core/profile_manager.py

```
@@ -164,20 +173,56 @@ def migrate_single_config_to_default() -> None:
+                default_cfg.unlink(missing_ok=True)
+                return
             bak = base_dir / "config.toml.pre-profile.bak"
             old_config.rename(bak)
-        except Exception:
-            pass
+            _log.info(
+                "Migrated legacy config %s -> %s",
+                old_config,
+                default_cfg,
+            )
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)